qa/integration/specs/cli/install_spec.rb

```diff
@@ -152,4 +162,95 @@ def gem_in_lock_file?(pattern, lock_file)
       end
     end
   end
+
+  context "rubygems hosted plugin" do
+    include_context "pluginmanager validation helpers"
+    shared_examples("overwriting existing") do
+      # let(:plugin_name) { defined?(super) ? super() : fail }
+      # let(:existing_plugin_version) { defined?(super) ? super() : fail }
+      # let(:specified_plugin_version) { defined?(super) ? super() : fail }
+      before(:each) do
```

Comment:
```suggestion
      before(:each) do
```
Remove commented out bindings